geocity/apps/permits/admin.py

```diff
@@ -514,6 +516,12 @@ def get__is_backoffice(self, obj):
     get__is_backoffice.admin_order_field = "permitdepartment__is_backoffice"
     get__is_backoffice.short_description = _("Secrétariat")
 
+    def get__duo_config(self, obj):
```

Comment:
Only show when DUO field is Mandatory 2FA is checked ?

Reply:
Done